src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -5,270 +5,288 @@
-                    class="mt-4 mb-n6"
-                    :editing="true"
-                    :schema="PersonAddressSchema"
-                    :address="inProgressMailingAddress"
-                    @update:address="updateMailingAddress"
-                    @valid="updateMailingAddressValidity"
-                  />
-                </article>
-
-                <!-- Delivery Address (for all roles) -->
-                <v-checkbox
-                  class="inherit-checkbox mt-8"
-                  hide-details
-                  label="Delivery Address same as Mailing Address"
-                  v-model="inheritMailingAddress"
+                  </v-col>
+                </v-row>
+              </v-card>
+            </article>
+
+            <!-- Business Number (for proprietors only) -->
+            <article v-if="isPerson && isProprietor" class="mt-8">
+              <label>Business Number</label>
+              <p class="mt-4 mb-0">
+                If you have an existing business number, enter it below and we will contact
+                Canada Revenue Agency and ask them to link it to this registration.
+              </p>
+              <HelpBusinessNumber class="mt-4" />
+              <v-text-field
+                filled
+                persistent-hint
+                class="item mt-4 mb-n2"
+                label="Business Number (Optional)"
+                id="person__business-number"
+                hint="First 9 digits of the CRA Business Number"
+                v-model.trim="orgPerson.officer.businessNumber"
+                :rules="BusinessNumberRules"
+              />
+            </article>
+
+            <!-- Email Address -->
+            <article class="mt-8">
+              <label>Email Address</label>
+              <p class="mt-4 mb-0">
+                Copies of the registration documents will be sent to this email address.
+              </p>
+              <v-text-field
+                filled
+                class="item mt-4 mb-n6"
+                label="Email Address"
+                id="person__email-address"
+                v-model.trim="orgPerson.officer.email"
+                :rules="Rules.EmailRules"
+                :readonly="isCompletingParty && !isRoleStaff"
+              />
+            </article>
+
+            <!-- Mailing Address -->
+            <article class="mt-8">
+              <label>Mailing Address</label>
+              <MailingAddress
+                ref="mailingAddressNew"
+                class="mt-4 mb-n6"
+                :editing="true"
+                :schema="PersonAddressSchema"
+                :address="inProgressMailingAddress"
+                @update:address="updateMailingAddress($event)"
+                @valid="updateMailingAddressValidity($event)"
+              />
+              <!-- dummy component to make form invalid if mailing address is invalid -->
+              <v-input class="d-none" :rules="[() => mailingAddressValid]" />
```

Comment:
I tried using `v-validate` and `vuelidate` to do this, and this workaround ended up being the simplest (and the one that worked).

I created [11737](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11737) to update BaseAddress and no longer need this workaround.